DQM/SiStripMonitorClient/python/pset4GenericTriggerEventFlag_cfi.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+
+genericTriggerEventFlag4HLTdb = cms.PSet(
+   andOr         = cms.bool( False ),
+   dbLabel       = cms.string("SiStripDQMTrigger"), # ("TrackerDQMTrigger"),
+   andOrHlt      = cms.bool(True), # True:=OR; False:=AND
+   hltInputTag   = cms.InputTag( "TriggerResults::HLT" ),
+   hltPaths      = cms.vstring(""), #cms.vstring("HLT_ZeroBias_v*","HLT_HIZeroBias_v*")
+   hltDBKey      = cms.string("SiStrip_HLT"),
+   errorReplyHlt = cms.bool( False ),
+   verbosityLevel = cms.uint32(1)
```

Comment:
@mtosi I realised only now that the original `TrackerDQMTrigger` label in GT would have been more appropriate. At any rate. I am updating the GTs with the current label. 
In the next iteration (after this PR is merged) I'll:
1) remove the "SiStripDQMTrigger" label from GT
2) add the "TrackerDQMTrigger" label with the same payload 
3) change your configuration code.
At that point it will be a technical only change (therefore with 0 expected changes).
Does it sound like a plan?
Marco